### ISSUE TITLE
[BUG]注册软件，出现 HTTP Error 503错误

### ISSUE THREAD

**Issue by wsxhd7881** [dashingsoft/pyarmor#975]
注册软件，出现 HTTP Error 503错误。降级到pyarmor7版本，注册命令pyarmor reg不生效。升级到pyarmor8版本，注册也不成功,出现 HTTP Error 503。windows和Linux都有这个问题，防火墙也都关了。需要怎么操作解决这个问题，顺利注册呢？

D:\Program Files\Python\Python37\Scripts>pyarmor reg -p "CTDI" pyarmor-regcode-4001.txt
INFO     Python 3.7.9
INFO     Pyarmor 8.0 (trial), 000000, non-profits
INFO     prepare "pyarmor-regcode-4001.txt"
INFO     query key file from server
This license registration information will be

License Type    : pyarmor-basic
。。。。。。

Are you sure to continue? (yes/no) yes
INFO     process activation file "pyarmor-regcode-4001.txt"
INFO     send register request to server
CRITICAL unknown error, please check pyarmor.error.log
ERROR    HTTP Error 503: Service Temporarily Unavailable

**Comment by jondy** [comment]
这个文件 `pyarmor.error.log` 的内容是什么？

**Comment by jondy** [comment]
还有注册码是新购买的还是原来的？原来的是无法使用的

**Comment by fO-000** [comment]
我也遇到了这个问题：

```txt
$ pyarmor reg --upgrade --product "xxx" assets/pyarmor-regcode-xxxx.txt
INFO     Python 3.11.2
INFO     Pyarmor 8.0 (trial), 000000, non-profits
INFO     prepare "assets/pyarmor-regcode-xxxx.txt"
INFO     query key file from server

You are about to upgrade old Pyarmor license to Pyarmor Basic
License for Pyarmor 8.0+

The original license no: pyarmor-vax-xxxxxx

The upgraded license information will be

License Type    : pyarmor-old
License Owner   : xxx
Bind Product    : xxx


Are you sure to continue? (yes/no) yes
INFO     process upgrading file "assets/pyarmor-regcode-xxxx.txt"
INFO     old license no: pyarmor-vax-xxxxxx
INFO     send upgrade request to server
CRITICAL unknown error, please check pyarmor.error.log
ERROR    HTTP Error 503: Service Temporarily Unavailable
```

`pyarmor.error.log` 的内容如下：

```txt
53805 MainProcess 2023-03-15 20:40:35,046
Traceback (most recent call last):
  File "/home/x/.local/lib/python3.11/site-packages/pyarmor/cli/__main__.py", line 559, in main
    main_entry(sys.argv[1:])
  File "/home/x/.local/lib/python3.11/site-packages/pyarmor/cli/__main__.py", line 547, in main_entry
    args.func(ctx, args)
  File "/home/x/.local/lib/python3.11/site-packages/pyarmor/cli/__main__.py", line 184, in cmd_reg
    getattr(regsvr, meth)(regfile, args.product)
  File "/home/x/.local/lib/python3.11/site-packages/pyarmor/cli/register.py", line 281, in upgrade
    res = self._send_request(url)
          ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/x/.local/lib/python3.11/site-packages/pyarmor/cli/register.py", line 216, in _send_request
    return urlopen(url, None, timeout, context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/urllib/request.py", line 216, in urlopen
    return opener.open(url, data, timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/urllib/request.py", line 525, in open
    response = meth(req, response)
               ^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/urllib/request.py", line 634, in http_response
    response = self.parent.error(
               ^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/urllib/request.py", line 563, in error
    return self._call_chain(*args)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.11/urllib/request.py", line 496, in _call_chain
    result = func(*args)
             ^^^^^^^^^^^
  File "/usr/lib/python3.11/urllib/request.py", line 643, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 503: Service Temporarily Unavailable
```

**Comment by wsxhd7881** [comment]
> rror.log
授权注册码是今年3.9日购买的，一直无法正常注册。没有频繁注册，每天就尝试注册一两次。pyarmor.error.log信息如下：

10356 MainProcess 2023-03-16 08:19:27,514
Traceback (most recent call last):
  File "d:\program files\python\python37\lib\site-packages\pyarmor\cli\__main__.py", line 559, in main
    main_entry(sys.argv[1:])
  File "d:\program files\python\python37\lib\site-packages\pyarmor\cli\__main__.py", line 547, in main_entry
    args.func(ctx, args)
  File "d:\program files\python\python37\lib\site-packages\pyarmor\cli\__main__.py", line 184, in cmd_reg
    getattr(regsvr, meth)(regfile, args.product)
  File "d:\program files\python\python37\lib\site-packages\pyarmor\cli\register.py", line 305, in register
    res = self._send_request(url)
  File "d:\program files\python\python37\lib\site-packages\pyarmor\cli\register.py", line 216, in _send_request
    return urlopen(url, None, timeout, context=context)
  File "d:\program files\python\python37\lib\urllib\request.py", line 222, in urlopen
    return opener.open(url, data, timeout)
  File "d:\program files\python\python37\lib\urllib\request.py", line 531, in open
    response = meth(req, response)
  File "d:\program files\python\python37\lib\urllib\request.py", line 641, in http_response
    'http', request, response, code, msg, hdrs)
  File "d:\program files\python\python37\lib\urllib\request.py", line 569, in error
    return self._call_chain(*args)
  File "d:\program files\python\python37\lib\urllib\request.py", line 503, in _call_chain
    result = func(*args)
  File "d:\program files\python\python37\lib\urllib\request.py", line 649, in http_error_default
    raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 503: Service Temporarily Unavailable

**Comment by jondy** [comment]
> HTTP Error 503: Service Temporarily Unavailable

This issue has been fixed in 8.0.5
